var/spack/repos/builtin/packages/py-horton/package.py

```diff
@@ -0,0 +1,182 @@
+
+    version('2.1.1', sha256='4b3f87920d881030ba80f097326a744de2cfee5316aa4499cc9a6501f64b5060')
+
+    depends_on('python@2.7:2.99', type=('build', 'run'))
+    # note: The py-setuptools dep should be type='build', but build_systems/python.py
+    # adds "args += ['--single-version-externally-managed', '--root=/']"
+    # This conflicts with the present setuptools version (required by python2).
+    # So, it's hacked to run-dep here.
+    depends_on('py-setuptools@:44.1.0',   type='run')
+    depends_on('py-nose@1.1.2:',        type=('build'))
+    depends_on('py-numpy@1.9.1:1.16', type=('build', 'run'))
+    depends_on('py-scipy@0.11.0:1.2.3', type=('build', 'run'))
+    depends_on('py-cython@0.24.1:', type=('build', 'run'))
+    depends_on('py-h5py@2.2.1:2.10.0', type=('build', 'run'))
+    depends_on('py-matplotlib@1.0:2.2.5', type=('build', 'run'))
+    depends_on('libxc@2.2.2:',  type=('build', 'run'))
+    depends_on('libint@2.0.3: tune=cp2k-lmax-7', type=('build', 'run'))
+    depends_on('curl@7.0:',     type=('build', 'run'))
+
+    patch("cext.patch")
+
+    @run_before('build')
+    def gen_setup_cfg(self):
+        spec = self.spec
+        numpy = self.spec['py-numpy']
+        print("TODO: use numpy to create initial setup.cfg")
+        # print(dir(numpy.package))
+
+        self.set_blas_lapack(numpy.package)
+
+        def write_library_dirs(f, dirs):
+            f.write('library_dirs = {0}\n'.format(dirs))
+            if not ((platform.system() == 'Darwin') and
+                    (Version(platform.mac_ver()[0]).up_to(2) == Version(
+                        '10.12'))):
+                # f.write('rpath = {0}\n'.format(dirs))
+                f.write('extra_link_args = -Wl,-rpath={0}\n'.format(dirs))
+
+        deps = {'libxc':  {'name': 'libxc', 'libraries': 'xc'},
+                'libint': {'name': 'libint2', 'libraries': 'int2'}}
+        for key, val in deps.items():
+            # val['libraries'] = ','.join(spec[key].libs.names)
+            val['library_dirs'] = spec[key].prefix + '/lib'
+            val['include_dirs'] = spec[key].prefix + '/include'
+
+        # Tell numpy where to find BLAS/LAPACK libraries
+        with open('setup.cfg', 'w') as f:
+            with open('site.cfg') as site:
+                for line in site:
+                    s = line.strip()
+                    if s[0] == '[' and s[-1] == ']':
+                        f.write('[blas]\n')
+                    else:
+                        f.write(line)
+            for key, val in deps.items():
+                f.write('[{0}]\n'.format(val['name']))
+                f.write('libraries = {0}\n'.format(val['libraries']))
+                write_library_dirs(f, val['library_dirs'])
+                f.write('include_dirs = {0}\n'.format(val['include_dirs']))
+                if key == 'libint':
+                    f.write('extra_compile_args = -DLIBINT2_MAX_AM_ERI=7')
+
+    # copied from py-numpy
+    def set_blas_lapack(other, self):
```

Comment:
I wonder if we could prevent some of this code duplication by copying the `site.cfg` that is installed by `py-numpy`:
```
$ find . -name site.cfg
./lib/python3.8/site-packages/numpy/distutils/site.cfg
```
@becker33 @scheibelp @alalazo is there any way to call the method from the numpy package instead?

Reply:
I removed this code, since this (later) version of horton strangely removed its dependence on optimized blas.